chromium/browser_type.go

```diff
@@ -371,15 +371,16 @@ func execute(
 	// can run into a data race.
 	err = cmd.Start()
 	if os.IsNotExist(err) {
-		return nil, nil, fmt.Errorf("does not exist: %s", path)
+		return nil, nil, fmt.Errorf("file does not exist: %s", path)
 	}
 	if err != nil {
 		return nil, nil, fmt.Errorf("%w", err)
```

Comment:
```suggestion
		return nil, nil, err
```

There are several of these, but i think we should just be able to return the error without wrapping it, no?

Reply:
@ankur, that would result in a linter error `wrapCheck`.

Reply:
Ah, of course. Do we need that check?

Reply:
Yeah, that's why the empty wrapping. I think we should leave that linter enabled, yes.